frontend/src/Student/pages/Dashboard.js

```diff
@@ -7,7 +7,7 @@ const Dashboard = () => (
   <Container>
     <Row>
       <Col>
-        <Shows />
+        <Shows studentUsername='user3'/>
```

Comment:
I think you need to remove the username here.

Reply:
Oops, Good catch